docs/assemblies/proc_deploying-in-disconnected-environments.adoc

```diff
@@ -0,0 +1,50 @@
+[id="proc_deploying-in-disconnected-environments"]
+= Deploying OpenStack in a disconnected environment
+
+[role="_abstract"]
+== Process
+Deploying in disconnected environments can be achieved largely by following the OpenShift documentation for mirroring OLM Operators: https://docs.openshift.com/container-platform/4.17/disconnected/mirroring/installing-mirroring-installation-images.html#olm-mirror-catalog_installing-mirroring-installation-images
```

Comment:
link to 4.16 instead?

Reply:
should probably be docs.redhat.com as well, since the docs are moving there:

https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html-single/disconnected_installation_mirroring/index#olm-mirror-catalog_installing-mirroring-installation-images

Reply:
Oh very true